libvcx/src/v3/handlers/proof_presentation/prover/prover.rs

```diff
@@ -34,6 +35,9 @@ impl Prover {
     pub fn retrieve_credentials(&self) -> VcxResult<String> {
         trace!("Prover::retrieve_credentials >>>");
         let presentation_request = self.prover_sm.presentation_request().request_presentations_attach.content()?;
+        if settings::indy_mocks_enabled() {
+            return Ok(CREDS_FROM_PROOF_REQ.to_string());
```

Comment:
Which tests do require this?

Reply:
Using bit of a new approach here, I am not mocking this explicitly via `set_mock_creds_retrieved_for_proof_request`